oh_staff_ui/classes/OralHistoryMods.py

```diff
@@ -177,12 +177,12 @@ def _create_relateditem_audio(self, mi: MediaFile) -> MODSv34:
             ri.toc = TableOfContents(text=toc.value)
 
         for ts in MediaFile.objects.filter(
-            item=pi, file_type__file_code="text_master_index"
+            item=pi, file_type__file_code="text_master_transcript"
         ):
-            if ts.file_url != "":
-                ri.locations.append(
-                    LocationOH(url=ts.file_url, usage="primary display")
-                )
+            if ts.file_url != "" and ts.file_url.endswith(".xml"):
```

Comment:
Please add a test to cover this.  We already have `test_timed_log_attribute_is_added()` which covers the `text_master_transcript` case, but should have a new test to confirm that `text_master_index` files do *not* cause the addition of the timed log attribute.

Reply:
Added `test_absence_of_timed_log_with_text_master_index` test to confirm behavior when **only** a `text_master_index` is present